community-edition/generate_script/hcce.yam

```diff
@@ -338,13 +338,13 @@ data:
 
     [ret."Elixir.RetWeb.Plugs.AddCSP"]
```

Comment:
We might want to add a comment explaining that the following values become part of the Content Security Policy, and that URLs local to the cluster do not need to be added.